Removes NanoTrasen

FearTheBlackout opened the pull request:
🆑 Devildabeast
rscdel: After numerous lawsuits, NanoTrasen has been barred from serving as the primary research company aboard the SEV Torch. The Research Division will now be taken up by the ship's myriad contractors.
/🆑
NanoTrasen has now been replaced with either "corporate" or "science/research" in most grammatical instances.
The Security Guard and Corporate Pilot now wear generic versions of their previous uniforms with references to NT removed.
NanoTrasen Regulations have been removed until such time as they can be reworked.
NT awards have been removed and are now generic corporate medals to be worn by all contractors.
All Research roles are now contractors, save for the Corporate Liaison, who is civilian.
Objects that spawn on ship have been altered to remove references to NT.
Corporate Liaison now represents business in general and can advocate for any corporation they desire. They even get a fancy new badge that displays the character's faction!
Research doors are now purple Science-colored again. Other objects like headsets to be recolored at a later date.

Crushtoe commented:
no

IDTia commented:
no

dryerlint commented:
hey put me in the reddit screenshot

PurpleMartinJCK commented:
@Crushtoe @IDTia why not?

Crushtoe commented:
I like having NT as a single consistent entity that actually has any solid power, and not a bunch of contractors who individually can do jack when compared to the might of SolGov.

LorenLuke commented:
Contentious tag?

Nerev4r commented:
The end of the sovereignity meme rises.

HeyBanditoz commented:
If we're going to do this (which we shouldn't,) can we put it through a two week test?

chaoko99 commented:
Absolutely not. I like in-round conflict between two entities. Sometimes it's nice to talk to people rather than fucking governmental pieheads who want to toss you into the brig and abandon you instead of roleplaying. Nonmil has a certain humanism that others lack.

ThatOneGuy42 commented:
100% for this with some other changes alongside it tho

Nerev4r commented:
This doesn't remove non-mil. This removes NanoTrasen.

EcklesFire commented:
Honestly if we're redoing science then we should get rid of the guard job, make it an EC department with an EC head of staff, have EC scientists along with contractor scientists and make the pilot the "shuttle pilot" instead. 

Or something like that. This? It's uh, not really very good, imho.

HeyBanditoz commented:
@ThatOneGuy42 Can you go into more detail?

IDTia commented:
If you trial it pair it with the removal of marines and fleet.

ThatOneGuy42 commented:
Oh fuck I had to read through this in more detail.

I’m 100% behind removing NT. This does not do justice to Science department though. Make it open to EC and contractors, kill NT sec and NT pilot and NT rep(replaced with contractor rep or something from FTU) and there is a few documents done on how to properly set up the ship like this. I’ll do a post tonight/tomorrow.

ghost commented:
didn't people want a department unbesmirched by the government or something

because i'm not going to put them in

chaoko99 commented:
@ParadoxSpace Read my post again. This time, read it.

AtomicWorm commented:
mom get the camera

HeyBanditoz commented:
Seriously. Where's the contentious tag in this? Does hot cover it?

babydoll commented:
Nobody is going to come in here and speed merge this, calm down. The tags do absolutely nothing mechanically.

HeyBanditoz commented:
I'm pretty sure contentious allows this PR to be discussed on the forums.

babydoll commented:
Well, that's only if a developer wants it to be; and I think the discussion on this subject has already happened about 50 times over the course of the past few months, and anyone's opinion on it is surely known.

Zero-Bits commented:
For once, I agree with ToG. But, I would like NT to remain as a corp option for contractors. But that's just a personal request.

Maveriknight commented:
Great, so instead of one solid corporation working as research on the torch, we'd have a legitimized splintered dozen or so factions researching there instead, all supposedly being meant to answer to one guy who probably will be working for yet another random corporation of their choosing, ultimately making research untameable as a whole, with a million agendas by a dozen different people at a time only tenuously held together under the banner of research.

PurpleMartinJCK commented:
This is awaiting lore and needs to have a conversation in the discussion and suggestions forum, NOT the gameplay forum. Wait for a topic to be put up there by @FearTheBlackout and then discuss it there. As usual, we welcome any code critique or reviews.

AtomicWorm commented:
On an actually serious comment this time, I've always wanted to get rid of NT simply because of the "muh sovereignity" meme. But splitting it into contractors from all over the place is just not good. I'd say make the RD and scientists part of the Expeditionary Corps, while having all scientist slots able to be contractors, including NT contractors.

Chiquiii commented:
If we are going to do this then i agree with that broseph is suggesting. Transfer research over to the EC and allow players to play civilians who have been hired by the EC or are government scientists which can allow them to be part of the EC yet not be tied up by milrp; as well as adding in contractors and such.

comma commented:
Contentious tag is for gameplay changes.
This is lore,
Chillax, 'needs lore' is an eternal blocker tag, unless it's tajaran swag.

ghost commented:
# **No.**

(Before _somebody_ asks "why", simply because I enjoy playing NT.)

PurpleMartinJCK commented:
Again, stop the lore talk here and wait until a forum thing is posted.

babydoll commented on code/modules/clothing/under/accessories/badges.dm:
```diff
@@ -120,14 +120,28 @@
+	badge_string = ""
+
+/obj/item/clothing/accessory/badge/corporate/proc/set_faction(var/faction)
+    badge_string = "[faction]"
```
This should probably check if the faction is set to Unset or None and then give them something like 'Unaffiliated', or inform them they can't use it without a faction.

Minijar commented:
No this is terrible. Research was supposed to be the last safe haven of civillian only players. SHoving it under EC or 'vague contractor' is a terrible idea. Having NT aboard, allows a major faction that can oppose and cause conflict with Sol Gov forces. 

I'm fairly certain someone promised somewhere that NT would stay no matter what. Could be wrong there though.

babydoll commented:
![](https://i.imgur.com/6jsk1Ig.png)

Techhead0 commented:
If was to change this, I'd probably have a single non-NT organization running science and push Pathfinder/Explorer into science under them.

Chiquiii commented:
We might want to lock this for now until a thread gets posted.

Sbotkin commented:
Waiting for "remove all branches except Fleet" PR.

(also this is obviously a meme PR, as well as the remove marines and fleet from comma)

Hubblenaut commented:
There's currently polls still being run and proposals being worked on as well as future, long-term goals being set for the server.
Is it really too much to ask to wait until the community has settled on a common goal to persue and the polls are fully evaluated instead of opening severely game-changing PRs independent from all that? I can't even find a forum-thread for this to properly discuss these changes in.

ghost commented:
#fakenews

AtomicWorm commented:
A reminder that I've been warned by Techhead for opening PRs as a means of doing activism. If this is one such case, I'd advise against it.

Hubblenaut commented:
>#fakenews

In that case, good job on wasting everybody's time, I guess.

ghost commented:
oops

in more serious news I have no opinion on removing NT. this is just how I envisioned it if we were to go through with it

Commit:
Removes NanoTrasen

Update jobs.dm
Update outfits.dm
Update access.dm
Update cards_ids.dm
Update encryption_keys.dm
Update headsets.dm
Update manuals.dm
Update stamps.dm
Update science.dm
Update loadout_accessories.dm
Update _defines.dm
Update headsets.dm
Update research.dm
Update misc.dm
Update torch_areas.dm
Merge branch 'dev' into feartheblackout/dieNTdie
Revert "Merge branch 'dev' into feartheblackout/dieNTdie"

This reverts commit e170ecb19b32b43589de8c7df05d2a83272b98fd.

Disarms line officers

'Self-defense' hahaha.

Pistols are gone from CE/CMO's lockers.
CoS's pistol is replaced with egun.
Mine overmap spacemove icon proc optimizations

Unga bunga me use CHECK_TICK
Typeless loops too as I heard they help
Merge remote-tracking branch 'origin/feartheblackout/dieNTdie' into feartheblackout/dieNTdie


Disarms line officers

'Self-defense' hahaha.

Pistols are gone from CE/CMO's lockers.
CoS's pistol is replaced with egun.
Mine overmap spacemove icon proc optimizations

Unga bunga me use CHECK_TICK
Typeless loops too as I heard they help

HeyBanditoz commented:
Food for thought:
![Food for thought](https://i.imgur.com/VA6b5UI.png)

ghost commented:
Obligatory "_votes do not matter_".

comma commented:
>I have no opinion on removing NT

Don't open PRs for changes you don't want to go through then

ghost commented:
Having no opinion is not the same as not wanting something :|

also I'm still working on this

HeyBanditoz commented:
Just keep in mind the vast majority doesn't want this.

On January 17, 2018 1:02:05 PM MST, FearTheBlackout <notifications@github.com> wrote:
>Having no opinion is not the same as not wanting something :|
>
>-- 
>You are receiving this because you commented.
>Reply to this email directly or view it on GitHub:
>https://github.com/Baystation12/Baystation12/pull/20282#issuecomment-358426444

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.

ghost commented:
doesn't matter if it gets approval from the relevant people 🤷

I'd like this reopened as I was going to start working on TOG's suggestions

Techhead0 commented:
A reminder that development is not just a popularity contest, especially not one voted on by GitHub reactions.

@FearTheBlackout At this point, I'd just work on TOG's suggestions locally and open a new PR when it's done and ready, or close to.
